langwatch/src/server/api/routers/evaluations.ts

```diff
@@ -4,25 +4,50 @@ import { TeamRoleGroup, checkUserPermissionForProject } from "../permission";
 import { evaluatorsSchema } from "../../evaluations/evaluators.zod.generated";
 import { runEvaluationForTrace } from "../../background/workers/evaluationsWorker";
 import { AVAILABLE_EVALUATORS } from "../../evaluations/evaluators.generated";
+import { prisma } from "~/server/db";
 
 export const evaluationsRouter = createTRPCRouter({
   availableEvaluators: protectedProcedure
     .input(z.object({ projectId: z.string() }))
     .use(checkUserPermissionForProject(TeamRoleGroup.GUARDRAILS_MANAGE))
     .query(async () => {
       return Object.fromEntries(
-        Object.entries(AVAILABLE_EVALUATORS)
-          .map(([key, evaluator]) => [
-            key,
-            {
-              ...evaluator,
-              missingEnvVars: evaluator.envVars.filter(
-                (envVar) => !process.env[envVar]
-              ),
-            },
-          ])
+        Object.entries(AVAILABLE_EVALUATORS).map(([key, evaluator]) => [
+          key,
+          {
+            ...evaluator,
+            missingEnvVars: evaluator.envVars.filter(
+              (envVar) => !process.env[envVar]
+            ),
+          },
+        ])
       );
     }),
+
+  availableCustomEvaluators: protectedProcedure
+    .input(z.object({ projectId: z.string() }))
+    .use(checkUserPermissionForProject(TeamRoleGroup.GUARDRAILS_MANAGE))
```

Comment:
not related to your PR, but probably we should rename those permissions to EVALUATIONS_VIEW and EVALUATIONS_MANAGE, don't know why I created it with guardrails in mind